src/NServiceBus.Persistence.AzureStorage/Config/CloudTableClientForSubscriptionsFromConnectionString.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NServiceBus.Persistence.AzureStorage
+{
+    using Microsoft.Azure.Cosmos.Table;
+
+    class CloudTableClientForSubscriptionsFromConnectionString : IProvideCloudTableClientForSubscriptions
+    {
+        public CloudTableClientForSubscriptionsFromConnectionString(string subscriptionConnectionString)
+        {
+            var subscriptionAccount = CloudStorageAccount.Parse(subscriptionConnectionString);
+            Client = subscriptionAccount.CreateCloudTableClient();
+            Client.DefaultRequestOptions = new TableRequestOptions
+            {
+                RetryPolicy = new ExponentialRetry()
```

Comment:
There's a discrepancy between Azure databus and ASP. Databus used the `Fixed` retry policy ([link](https://github.com/Particular/NServiceBus.DataBus.AzureBlobStorage/blob/8c2980589a7a929d2a7a3ce1f52544c2c9d3e095/src/DataBus/AzureDataBusPersistence.cs#L52)).

Reply:
I know but this is the one we previously had so I left it compliant with what we had before